### DISCUSSION TITLE
Question about "PrismLibrary/Prism-Samples-Wpf/07-Modules-Xaml"

### DISCUSSION THREAD

**Discussion by YJ15** [PrismLibrary/Prism#2371] [WPF]
When I test "PrismLibrary/Prism-Samples-Wpf/07-Modules-Xaml" I get the exception below.

System.Windows.Markup.XamlParseException (hex value 0x0C is an invalid character. Line 1, position 1')
Inner Exception XmlException (hex value 0x0C is an invalid character. Line 1, position 1)

Can I get help?
※ Environment: Visual Studio 2017, .NET Framework 4.6.1, Windows 7
※ Error is not caught when try-catch is used in 'new XamlModuleCatalog'. The return value of 'new XamlModuleCatalog' is Prism.Modularity.XamlModuleCatalog and its item count is zero.

**Comment by brianlagunas** [comment]
after updating the code to 
```
        protected override IModuleCatalog CreateModuleCatalog()
        {
            return new XamlModuleCatalog(new Uri("/Modules;component/ModuleCatalog.xaml", UriKind.Relative));
        }
```
it works fine for me.

**Reply by YJ15** [reply]
Thank you for the reply. But my test code is the same as yours.

### App.xaml.cs
```
protected override IModuleCatalog CreateModuleCatalog()
{
    return new XamlModuleCatalog(new Uri("/Modules;component/ModuleCatalog.xaml", UriKind.Relative));
}
```

### ModuleCatalog.xaml
```
<m:ModuleCatalog xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"    
                    xmlns:m="clr-namespace:Prism.Modularity;assembly=Prism.Wpf">

    <m:ModuleInfo ModuleName="ModuleAModule" 
                  ModuleType="ModuleA.ModuleAModule, ModuleA, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"/>

</m:ModuleCatalog>
```
